tests/Handler/InputHandlerTest.php

```diff
@@ -5,18 +5,39 @@
 use OckCyp\CoversValidator\Handler\InputHandler;
 use OckCyp\CoversValidator\Model\ConfigurationHolder;
 use OckCyp\CoversValidator\Tests\BaseTestCase;
+use Prophecy\Prophet;
 
 /**
  * @coversDefaultClass \OckCyp\CoversValidator\Handler\InputHandler
  */
 class InputHandlerTest extends BaseTestCase
 {
+    private $prophet;
+
+    protected function setUp(): void
+    {
+        if (!class_exists(Prophet::class)) {
+            $this->markTestSkipped('Skip Prophet-based tests on PHP 7.1');
+
+            return;
+        }
+
+        $this->prophet = new Prophet();
+    }
+
+    protected function tearDown(): void
+    {
+        if ($this->prophet) {
+            $this->prophet->checkPredictions();
+        }
+    }
+
     /**
      * @covers ::handleInput
      */
     public function testHandlesInputWithNoBootstrap()
     {
-        $input = $this->prophesize(
+        $input = $this->prophet->prophesize(
```

Comment:
is it possible to have a method that adapts to PHP 7.1 for this one? all it needs to do is to use `$this->prophesize()` vs `$this->prophet->prophesize()`